src/oncall/ui/templates/index.html

```diff
@@ -1159,46 +1159,31 @@ <h3>User Settings</h3>
             </select>
           </td>
         </tr>
+        {{#each contactmodes}}
         <tr>
-          <td>Phone</td>
+          <td>{{label}}</td>
           <td>
             <span>
-            {{#if contacts.call}}
-              <a href="tel:{{contacts.call}}"> {{contacts.call}} </a>
-            {{else}}
-              {% endraw %}
-                {{missing_number_note|safe}}
-              {% raw %}
-            {{/if}}
-            </span>
-          </td>
-        </tr>
-        <tr>
-          <td>SMS Number</td>
-          <td>
-            <span>
-              {{#if contacts.sms}}
-                <a href="tel:{{contacts.sms}}"> {{contacts.sms}} </a>
-              {{else}}
-                {% endraw %}
-                  {{missing_number_note|safe}}
-                {% raw %}
-              {{/if}}
+            {% endraw %} {% if usercontact_ui_readonly %} {% raw %}
+              {{#inArray mode ../telmodes}}
+                  {{#if value}}
+                    <a href="tel:{{value}}"> {{value}} </a>
+                  {{else}}
+                    {% endraw %}
+                      {{missing_number_note|safe}}
+                    {% raw %}
+                  {{/if}}
+              {{/inArray}}
+              {{^inArray mode ../telmodes}}
```

Comment:
I actually haven't seen this `{{^` syntax before for handlebars, and couldnt find anything on their site. Would you mind clarifying what this does?

Reply:
Indeed, there is little to no documentation on this. There is a single reference here: http://handlebarsjs.com/expressions.html. It inverts the helpers output. I'll submit a different variant on this which is more obvious.

Reply:
Ah, I thought I could simplify by doing:

```
{{inArray modes ../telmodes}}
  {{val}}
{{^}}
  {{altval}}
{{/inArray}}
```

But apparently that's not the way it works. Nor could I replace {{^}} with {{^inArray}}. So it remains as it for now.